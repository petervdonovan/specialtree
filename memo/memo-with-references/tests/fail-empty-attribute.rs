use memo_with_references::memo;

#[memo]
fn test_empty_attribute() {}

fn main() {}
