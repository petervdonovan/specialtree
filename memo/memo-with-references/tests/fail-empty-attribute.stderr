error: expected lifetime in #[memo('a)]
 --> tests/fail-empty-attribute.rs:3:1
  |
3 | #[memo]
  | ^^^^^^^
  |
  = note: this error originates in the attribute macro `memo` (in Nightly builds, run with -Z macro-backtrace for more info)
